h/migrations/versions/3e1727613916_add_document_web_uri_column.py

```diff
@@ -0,0 +1,24 @@
+"""
+Add document web_uri column
+
+Revision ID: 3e1727613916
+Revises: a001b7b4c78e
+Create Date: 2016-09-12 13:21:56.739838
+"""
+
+from __future__ import unicode_literals
+
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '3e1727613916'
+down_revision = 'a001b7b4c78e'
+
+
+def upgrade():
+    op.add_column('document', sa.Column('web_uri', sa.UnicodeText()))
```

Comment:
I'm also open to suggestions regarding the name `web_uri`. This will represent the first `http` or `https` URI that we know, so I thought `web_uri` is concise and descriptive enough.